shared/markdown/parser.pegjs

```diff
@@ -118,6 +120,14 @@ Channel
     options && options.channelNameToConvID && options.channelNameToConvID(name)
 } { return {type: 'channel', children: [name], convID: options && options.channelNameToConvID && options.channelNameToConvID(name) } }
 
+Num = [0-9]
+Phone
+ = phone:($ (
+   (PhoneMarker? Num Num Num PhonePostfix Num Num Num [- ] Num Num Num Num !NonBlank) /
+   (PhoneMarker? Num Num Num [- ] Num Num Num [- ] Num Num Num Num !NonBlank))) {
```

Comment:
Would it cause problems to add a variant `Num{10}Num?` to support phone numbers with no formatting?

Reply:
@buoyad Makes sense, done, PTAL!